app/templates/direct-award/index.html

```diff
@@ -5,22 +5,19 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with
-    items = [
-      {
-          "link": url_for('main.index'),
-          "label": "Digital Marketplace"
-      },
-      {
-          "link": url_for('external.buyer_dashboard'),
-          "label": "Your account"
-      }
-    ]
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
-{% endblock %}
+{{ govukBreadcrumbs({
+  "items": [
+    {
+      "href": url_for('main.index'),
+      "text": "Digital Marketplace"
+    },
+    {
+      "href": url_for('external.buyer_dashboard'),
+      "text": "Your account"
+    },
```

Comment:
I think we should have a final breadcrumb for this page

```suggestion
    },
    {
      "text": "Your saved searches"
    }
```

Reply:
Done 👍 

https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/965#issuecomment-565332887